hexer/Processor.cpp

```diff
@@ -82,6 +84,31 @@ void process(HexGrid *grid, PointReader reader)
     grid->findParentPaths();
 }
 
+void processLaz(HexGrid *grid, std::ifstream& file)
+{
+    char buf[30];
```

Comment:
Some LAS/LAZ files will have point records greater than 30 bytes. You'll have to get the size from the header and make a buffer that's the right size. In C++ you would normally use a std::vector as the buffer when you don't know the size until runtime. There are lots of examples of this around, but let me know if you have trouble.